app/assets/javascripts/admin/analytics-modules/ga4-button-setup.js

```diff
@@ -10,15 +10,12 @@ window.GOVUK.analyticsGa4.analyticsModules =
         "[data-module~='ga4-button-setup']"
       )
       moduleElements.forEach(function (moduleElement) {
-        const buttons = moduleElement.querySelectorAll(
-          'button'
```

Comment:
Nit: this reduction to one line should be fixed up to the first commit.

And the second commit should explain what you've explained in the PR description:
> There was a hardcoded "method" attribute which we unset, as we don't know whether it's actually a primary click or not. The link tracker from Publishing components does apply this attribute.

(NB, that last sentence reads a bit confusingly - I'd suggest rewording along the lines of "the link tracker from Publishing components already applies the attribute, so we don't need to repeat it here")